ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextViewManager.java

```diff
@@ -69,6 +69,21 @@ public void setTextAlign(ReactTextView view, @Nullable String textAlign) {
       throw new JSApplicationIllegalArgumentException("Invalid textAlign: " + textAlign);
     }
   }
+  
+  @ReactProp(name = "textDecorationLine")
+    public void setTextDecorationLine(ReactTextView view, @Nullable String textDecorationLine) {
+        if (textDecorationLine == null || "none".equals(textDecorationLine)) {
```

Comment:
We should preserve the other flags on the paint instead of assuming they are only ANTI_ALIAS_FLAG (use `textView.getPaint().getFlags()`). We also should unset any flags we set if text decoration is `"none"` or `null`.